### DISCUSSION TITLE
Is this what the update dialogue looks like for Sparkle?

### DISCUSSION THREAD

**Discussion by MoltenMonster** [sparkle-project/Sparkle#2415] [Q&A] [answered]
I'm curious because I've seen this in many macOS apps and I know Sparkle is a popular library for delivering app updates.
<img width="384" alt="Screenshot 2023-08-07 at 12 24 13 PM" src="https://github.com/sparkle-project/Sparkle/assets/42647030/164e6549-106e-40fa-aea0-2a14c4a5d412">

**Comment by zorgiepoo** [comment, marked as answer]
It looks like a version of Sparkle, yes.
